cht/adapter.py

```diff
@@ -2,307 +2,296 @@
-        self.precice.configure(preciceConfigFile)
-        self.configure(config)
-    
-    def configure(self, config):
-        L = [None] * self.numInterfaces        # Loads
-        SM = [None] * self.numInterfaces    # Shifted meshes
-        for i in range(self.numInterfaces):
-            # Shifted mesh (interface nodes displaced by a distance delta in the direction of the surface normal
-            # towards the inside of the solid)
-            SM[i] = CREA_MAILLAGE(MAILLAGE=self.MESH, RESTREINT={"GROUP_MA": config[i]["patch"], "GROUP_NO": config[i]["patch"]})
-            # Create interface
-            interface = Interface(self.precice, self.participantName, config[i], self.MESH, SM[i], self.MODEL, self.MAT[config[i]["material-id"]], self.isNonLinear)
-            # Loads
-            BCs = interface.createBCs()
-            L[i] = AFFE_CHAR_THER(MODELE=self.MODEL, ECHANGE=BCs)
-            interface.setLoad(L[i])
-            self.LOADS.append({'CHARGE': L[i]})
-            self.interfaces.append(interface)
-
-    def initialize(self, INIT_T):
-
-        self.preciceDt = self.precice.initialize()
-
-        if self.precice.isActionRequired(PyActionWriteInitialData()):
-            self.writeCouplingData(INIT_T)
-            self.precice.fulfilledAction(PyActionWriteInitialData())
-
-        self.precice.initializeData()
-
-        return self.preciceDt
-
-    def isCouplingOngoing(self):
-        return self.precice.isCouplingOngoing()
-    
-    def writeCouplingData(self, TEMP):
-        if self.precice.isWriteDataRequired(self.preciceDt):
-            for interface in self.interfaces:
-                interface.writeBCs(TEMP)
-    
-    def readCouplingData(self):
-        if self.precice.isReadDataAvailable():
-            for interface in self.interfaces:
-                interface.readAndUpdateBCs()
-
-    def writeCheckpoint(self):
-        if self.precice.isActionRequired(PyActionWriteIterationCheckpoint()):
-            # Do nothing
-            self.precice.fulfilledAction(PyActionWriteIterationCheckpoint())
-
-    def readCheckpoint(self):
-        if self.precice.isActionRequired(PyActionReadIterationCheckpoint()):
-            # Do nothing
-            self.precice.fulfilledAction(PyActionReadIterationCheckpoint())
-
-    def isCouplingTimestepComplete(self):
-        return self.precice.isTimestepComplete()
-
-    def advance(self):
-        self.preciceDt = self.precice.advance(self.preciceDt)
-        return self.preciceDt
-
-    def finalize(self):
-        self.precice.finalize()
+	def __init__(self, preciceConfigFile, participantName, config, MESH, MODEL, MAT, isNonLinear=False):
+		self.interfaces = []
+		self.numInterfaces = len(config)
+		self.MESH = MESH
+		self.MODEL = MODEL
+		self.MAT = MAT
+		self.LOADS = []
+		self.isNonLinear = isNonLinear
+		self.participantName = participantName
+		self.preciceDt = -1
+		self.precice = precice.Interface(participantName, 0, 1)
```

Comment:
Is there a reason for changing the indentation everywhere? If not, please keep the previous one.

Reply:
Python needs a correct indentation to run correctly. Somehow I got an error for incorrect indentation for the previous one, therefore I have redone the indentation of the entire code.

Reply:
It does need tabs, but somehow the size of tabs (in spaces) was changed. This is something you can configure in your editor (e.g. in Atom: `Edit > Preferences > Editor > Tab Length`).

Reply:
Please ignore my above comments and keep your indentation.